source/_patterns/01-atoms/dropdown/_dropdown.twig

```diff
@@ -0,0 +1,56 @@
+{#
+  Dropdown!
+  dropdown_button: Boolean: True|False: Choose whether the dropdown will be a button element or not.
+  other_classes: String: Allows use of other bootstrap utility classes.
+  dropup: Boolean: True|False: Set the dropdown to dropup instead.
+  dropdown_color: String: primary|secondary|success|danger|warning|info|light|dark: The color of the dropdown.
+  dropdown_split: Boolean: True|False: Set the dropdown button to split between text and the arrow.
+  dropdown_text: String: Set the dropdown's text.
+  dropdown_menu_right: Boolean: True| False: Set the menu to drop on the right side.
+  dropdown_element: String: a|button: Set the element of the menu items
+  dropdown_menu_aria_label: Do not set: Will set automatically if you assign an button_id
+  dropdown_menu_content: Block: Set content that will be within the dropdown menu.
+  dropdown_menu_items: Array: An array of menu items.
+  dropdown_menu_item_disabled: Boolean: True|False: Set an individual menu option as disabled.
+  dropdown_menu_item_text: String: Set the text of an individual menu item.
+  See https://getbootstrap.com/docs/4.0/components/dropdowns/ fro more details
+#}
+
+{% if dropdown_button %}
+  {% set dropdown_class = 'btn-group' %}
+{% else %}
+  {% set dropdown_class = 'dropdown' %}
+{% endif %}
+
+{% set other_classes = other_classes ?: other_classes %}
```

Comment:
Did @illepic mention swapping the `?:` format approach?

Reply:
yup, thanks for the reminder